app/electron/menu.js

```diff
@@ -195,6 +196,7 @@ const MenuBuilder = (mainWindow, appName) => {
           {
             label: "Learn More",
             click: async () => {
+              // eslint-disable-next-line global-require
```

Comment:
Do not disable eslint, let the warning be there, as a reminder that we have used global requires, we could refactor this to not require global require, but that might cause unnecessary  overhead when not used